src/Microsoft.Sbom.Parsers.Spdx22SbomParser/Utils/SPDXToSbomFormatConverterExtensions.cs

```diff
@@ -26,10 +26,6 @@ public static class SPDXToSbomFormatConverterExtensions
     public static SbomFile ToSbomFile(this SPDXFile spdxFile)
     {
         var checksums = spdxFile.FileChecksums?.Select(c => c.ToSbomChecksum());
-        if (!checksums.Any() || checksums.All(c => c.Algorithm != AlgorithmName.SHA256))
-        {
-            throw new ParserException("File hash is missing a SHA256 value");
```

Comment:
Before this change, code coverage shows that we're covering the throw but never getting past it. Based on that, should there be at least 1 unit test change?